packages/tupaia-web/src/features/Dashboard/DashboardMenu.tsx

```diff
@@ -4,21 +4,37 @@
  */
 import React, { useState } from 'react';
 import { useLocation, Link, useParams } from 'react-router-dom';
-import ArrowDropDownIcon from '@material-ui/icons/ArrowDropDown';
+import KeyboardArrowDownIcon from '@material-ui/icons/KeyboardArrowDown';
 import { ButtonBase, Menu, MenuItem } from '@material-ui/core';
 import styled from 'styled-components';
 import { Dashboard } from '../../types';
 
 const MenuButton = styled(ButtonBase)`
   display: flex;
-  justify-content: space-between;
-  align-items: center;
-  background-color: ${({ theme }) => theme.panel.secondaryBackground};
+  justify-content: flex-start;
+  align-items: flex-end;
+  background-color: #202124;
```

Comment:
Should be using a theme variable here, since where possible we want to avoid hard coded colours, if they're colours we use in multiple places